bar-trender-front-end/src/views/ErrorPage.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import DeviceIdentifier from 'react-device-identifier';
+
+// reactstrap components
+import {
+  Input,
+  InputGroupAddon,
+  InputGroupText,
+  InputGroup,
+  Container,
+  Row,
+  Col,
+} from "reactstrap";
+
+// core components
+import MainNavbar from "../components/Navbars/MainNavbar.js";
+import LandingPageHeader from "../components/Headers/LandingPageHeader.js";
+import DefaultFooter from "../components/Footers/DefaultFooter.js";
+
+// images
+import image_left from '../assets/img/expositions/hU-kQ3Epxeq2dhaBpUgYfYaPhHEOKXnHXSeUqLjTygYBV05OHhUSZEWilh_Da9zkI1d_cgz91KIPevD_BBhBWhaKevognkx6Bv7-QwkQdRG9oznKG6wOae4avH8ksi6bkJBLWl4.png';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7131439)
```suggestion
import image_left from "../assets/img/expositions/hU-kQ3Epxeq2dhaBpUgYfYaPhHEOKXnHXSeUqLjTygYBV05OHhUSZEWilh_Da9zkI1d_cgz91KIPevD_BBhBWhaKevognkx6Bv7-QwkQdRG9oznKG6wOae4avH8ksi6bkJBLWl4.png";
```